sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/models.py

```diff
@@ -548,85 +548,81 @@ def request_id(self):
 class CertificatePolicy(object):
     """Management policy for a certificate.
 
-    :param bool exportable: Indicates if the private key can be exported. For valid values,
+    :param str issuer_name: Name of the referenced issuer object or reserved names; for example,
+        'Self' or 'Unknown"
+    :keyword str subject_name: The subject name of the certificate. Should be a valid X509
+        distinguished name. Either subject_name or one of the subject alternative name parameters
+        are required.
+    :keyword Iterable[str] san_emails: Subject alternative emails of the X509 object. Only one out
+        of san_emails, san_dns_names, and san_upns may be set. Either subject_name or one of the
+        subject alternative name parameters are required.
+    :keyword Iterable[str] san_dns_names: Subject alternative DNS names of the X509 object. Only one out
+        of san_emails, san_dns_names, and san_upns may be set. Either subject_name or one of the
+        subject alternative name parameters are required.
+    :keyword Iterable[str] san_upns: Subject alternative user principal names of the X509 object. Only one out
+        of san_emails, san_dns_names, and san_upns may be set. Either subject_name or one of the
+        subject alternative name parameters are required.
+    :keyword bool exportable: Indicates if the private key can be exported. For valid values,
         see KeyType.
-    :param key_type: The type of key pair to be used for the certificate.
-    :type key_type: str or ~azure.keyvault.certificates.enums.KeyType
-    :param int key_size: The key size in bits. For example: 2048, 3072, or 4096
+    :keyword key_type: The type of key pair to be used for the certificate.
+    :paramtype key_type: str or ~azure.keyvault.certificates.enums.KeyType
+    :keyword int key_size: The key size in bits. For example: 2048, 3072, or 4096
         for RSA.
-    :param bool reuse_key: Indicates if the same key pair will be used on certificate
+    :keyword bool reuse_key: Indicates if the same key pair will be used on certificate
         renewal.
-    :param curve: Elliptic curve name. For valid values, see KeyCurveName.
-    :type curve: str or ~azure.keyvault.certificates.enums.KeyCurveName
-    :param ekus: The enhanced key usages.
-    :type ekus: list[str]
-    :param key_usage: List of key usages.
-    :type key_usage: list[str or ~azure.keyvault.certificates.enums.KeyUsageType]
-    :param content_type: The media type (MIME type) of the secret backing the certificate.
+    :keyword curve: Elliptic curve name. For valid values, see KeyCurveName.
```

Comment:
Suggest removing the second sentence because the `paramtype` covers it.

Reply:
removed, thanks!